script.matchcenter/resources/skins/default/720p/script-matchcenter-Livescores.xml

```diff
@@ -11,17 +11,18 @@
 		<effect type="fade" start="100" end="0" time="240" />
 	</animation>
 	<controls>
-		<control type="image">
+		<control type="image" id="32502">
 			<left>158</left>
 			<top>24</top>
 			<width>966</width>
 			<height>663</height>
+			<visible>IsEmpty(Window(home).Property(loading))</visible>
```

Comment:
By the way @phil65 do you know how I can resize a window depending on the presence (or not) of a scrollbar? Right now I am doing it manually in python but it would be much better if all that stuff could be achieved though xml. Unfortunately I couldn't manage to have conditionals on the window width